contrib/demo/prototype2-script/script

```diff
@@ -201,14 +218,14 @@ wait
 clear
 
 c "Now we're going to show our new workspaces APIs! Think of a workspace like your own Kubernetes apiserver, where you can create your own CRDs without worrying about conflicting with anyone else's."
-c "Let's start by creating a 'demo' workspace"
-pe "kubectl kcp workspace --token user-1-token create demo --use"
+c "We are in the demo workspace now. Let's create a new workspace called 'playground' as user-1 and switch to it."
+pe "kubectl kcp workspace --token user-1-token create playground --use"
```

Comment:
Will this work with the last RBAC additions ? There should be the right permissions created for the demo org at the root level (maybe I missed where it is create...)

Reply:
it's fixed